astropy/io/fits/tests/test_image.py

```diff
@@ -1243,7 +1243,7 @@ def test_comp_image_hcompression_1_invalid_data(self):
             name="SCI",
             compression_type="HCOMPRESS_1",
             quantize_level=16,
-            tile_size=[2, 10, 10],
+            tile_shape=(2, 10, 10),
```

Comment:
I think the old tile shape was actually incorrect as I think it might have been given in numpy order already (when comparing to the data shape)